verilog/tools/syntax/verilog_syntax_test.sh

```diff
@@ -171,7 +171,7 @@ All lexed tokens:
 (#"endmodule" @11-20: "endmodule")
 (#"<<\\\\n>>" @20-21: "
 ")
-(#"end of file" @21-21: "")
+(#\$end @21-21: "")
```

Comment:
This looks like an artifact from scaling back from bison 3.6.2 to 3.3.2.
I looked if it was simple to actually upgrade the rules_bazel to 3.6.2 and it seems there is a new dependency ('textstyle') starting from 3.4 which makes things harder to port.

Apart from obvious issues integrating it in environments that are using a newer version (possibly the `end of file` string can be changed for the test), are you aware of any issues performance or feature-wise which would still warrant investigating going to the 3.6.x release @fangism ?

Reply:
Fixed in https://github.com/google/verible/pull/728/commits/70db525c5c50120f121b464f52c893cb456afa0a